### PR TITLE
Remind users to set the default launcher in settings

### LINKED ISSUE #2889: [BUG] Can't set Lawnchair as the default launcher on Android 13
### Describe the bug

Can't set Lawnchair as the default launcher on Android 13, without the app chooser dialog showing.

### Steps to reproduce

Launch the `Lawnchair`, press the home button, and the Pixel launcher had replaced the launcher.

### Expected behavior

Showing the default app chooser dialog after pressing the home button.

### Screenshots

https://user-images.githubusercontent.com/10363352/187065139-4a8b689c-5457-47fc-853e-bbe2d986f7b6.mov

### Device information

- Device: Google Pixel emulator
- OS: Android 13
- App version: https://github.com/LawnchairLauncher/lawnchair/releases/tag/v12.1.0-alpha.4

### Additional context

Link #2847.

### PR DESCRIPTION
**Pull request by Goooler**
## Description

Closes #2889.

## Type of change

:x: General change (non-breaking change that doesn't fit the below categories like copyediting)
:white_check_mark: Bug fix (non-breaking change which fixes an issue)
:x: New feature (non-breaking change which adds functionality)
:x: Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: lawnchair/res/values/strings.xml [hunk 1]
```diff
@@ -19,6 +19,7 @@
     <string name="iconPackPackageDefault" translatable="false">""</string>
 
     <!-- PreferenceDashboard -->
+    <string name="set_default_launcher_tip">Warning: Lawnchair isn\'t your default launcher, tap this to set me as your default in system settings.</string>
```
Comment (SuperDragonXD) [inline lawnchair/res/values/strings.xml]:
```suggestion
    <string name="set_default_launcher_tip">Tap this card to set Lawnchair as your default launcher.</string>
```

File: lawnchair/src/app/lawnchair/ui/preferences/PreferencesDashboard.kt [hunk 2]
```diff
@@ -176,6 +189,27 @@ fun PreferencesDebugWarning() {
     }
 }
 
+@Composable
+fun PreferencesSetDefaultLauncherWarning() {
+    val context = LocalContext.current
+    Surface(
+        modifier = Modifier.padding(horizontal = 16.dp),
+        shape = MaterialTheme.shapes.large,
+        color = Material3Theme.colorScheme.errorContainer
```
Comment (SuperDragonXD) [inline lawnchair/src/app/lawnchair/ui/preferences/PreferencesDashboard.kt]: Error color seems a little too harsh, maybe something like `surface` or similar can work.

**Review by SuperDragonXD — CHANGES_REQUESTED** [review]
I'd recommend not making the text look like an "you **must** set Lawnchair as default", maybe make the text suggest doing it instead (as seen by my changes). 

Also maybe move it to About, add an option in the 3 dots toolbar (like Lawnchair 2), or make it look like a preference option (like Hyperion Launcher)

File: lawnchair/src/app/lawnchair/ui/preferences/PreferencesDashboard.kt (hunk 2 above)
Comment (Goooler) [reply lawnchair/src/app/lawnchair/ui/preferences/PreferencesDashboard.kt]: Yeah, I was seeking some colors like yellow but didn't decide to use which one.

**Comment by Goooler** [comment]
How about this?

File: lawnchair/src/app/lawnchair/ui/preferences/PreferencesDashboard.kt (hunk 2 above)
Comment (Goooler) [reply lawnchair/src/app/lawnchair/ui/preferences/PreferencesDashboard.kt]: cf609c07ffb93d59d62cb40e35c3e257c4da7f2d

**Review by SuperDragonXD — APPROVED** [review]
Sure, that looks fine.

**Pull request merged**